Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -103,14 +104,23 @@ IndicatorInfoProvider {
         debouncer.action = { [weak self] in self?.fetch(page: nil) }
     }
 
+    func didChangeSegment(sectionController: SearchBarSectionController, index: Int) {
+        switch index {
+        case 0: updateState(isOpen: true)
+        case 1: updateState(isOpen: false)
+        default: break
+        }
+    }
+
     // MARK: BaseListViewControllerHeaderDataSource
 
     func headerModel(for adapter: ListSwiftAdapter) -> ListSwiftPair {
         return ListSwiftPair.pair("header", { [weak self, previousSearchString] in
             SearchBarSectionController(
                 placeholder: Constants.Strings.search,
                 delegate: self,
-                query: previousSearchString
+                query: previousSearchString,
+                items: ["Open", "Closed"]
```

Comment:
Well localization of those would never be done here, it would be done before adding to the segment control. Aka, user passes in array of strings, we localize and add to control.